services/app/src/prop-types/player.js

```diff
@@ -4,5 +4,7 @@ import playerStatuses from '../constants/playerStatuses';
 export default PropTypes.shape({
   name: PropTypes.string,
   status: PropTypes.oneOf(Object.values(playerStatuses)),
+  // eslint-disable-next-line react/forbid-prop-types
```

Comment:
`color` is used in `TeamMember` prop types. Would it be possible to reuse same type here and get rid of this ignore?